CNV-27866: [Doc] expose the downwardMetrics through a virtio-serial channel (#74920)
openshift/openshift-docs: virt/monitoring/virt-exposing-downward-metrics.adoc

:_mod-docs-content-type: ASSEMBLY
[id="virt-exposing-downward-metrics"]
= Exposing downward metrics for virtual machines
include::_attributes/common-attributes.adoc[]
:context: virt-exposing-downward-metrics

toc::[]

> As an administrator, you can expose a limited set of host and virtual machine (VM) metrics to a guest VM by configuring a YAML file. The YAML file must include a `virtio-serial` port, a `downwardMetrics` device, and a `DownwardMetrics` feature gate. Enabling the feature gate helps to ensure additional protection in restricting viewing of the metrics results.

Reviewer: ```suggestion
As an administrator, you can expose a limited set of host and virtual machine (VM) metrics to a guest VM by configuring a YAML file. The YAML file must include a `downwardMetrics` device, and the `DownwardMetrics` feature gate has to be enabled. Enabling the feature gate helps to ensure additional protection in restricting viewing of the metrics results.
```

The sentence `Enabling the feature gate helps to ensure additional protection in restricting viewing of the metrics results.`, IMHO, may lead to users to believe that if the feature gate is enabled, a restricted view is presented, which is not true. If the feature gate is not enabled, the user will not able to get any metric (in fact, the user will not be able to create a VM requesting downwardMetrics at all). If enabled, the user will be able to see all downwardMetrics exposed.

The idea behind the feature gate is to help the admin to restrict if VMs will be able or not to pull the downwardMetrics. Therfore, I will rephrase as: 

`Disabling the feature gate helps to ensure additional protection in restricting viewing of the metrics results.`

:_mod-docs-content-type: ASSEMBLY
[id="virt-exposing-downward-metrics"]
= Exposing downward metrics for virtual machines
include::_attributes/common-attributes.adoc[]
:context: virt-exposing-downward-metrics

toc::[]

As an administrator, you can expose a limited set of host and virtual machine (VM) metrics to a guest VM by:

> . Enabling a `downwardMetrics` feature gate. Enabling the feature gate helps to ensure additional protection in restricting viewing of the metrics results

Reviewer: ```suggestion
. Enabling a `downwardMetrics` feature gate. Disabling the feature gate helps to ensure additional protection in restricting viewing of the metrics results
```

:_mod-docs-content-type: ASSEMBLY
[id="virt-exposing-downward-metrics"]
= Exposing downward metrics for virtual machines
include::_attributes/common-attributes.adoc[]
:context: virt-exposing-downward-metrics

toc::[]

As an administrator, you can expose a limited set of host and virtual machine (VM) metrics to a guest VM by first enabling a `downwardMetrics` feature gate and then configuring a `downwardMetrics` device.

Users can view the metrics results by using the command line or the `vm-dump-metrics tool`.

[id="virt-enabling-disabling-feature-gate"]
== Enabling or disabling the downwardMetrics feature gate

You can enable or disable the `downwardMetrics` feature gate by performing either of the following actions:

> * Editing the HyperConverged custom resource (CM) in your default editor

Reviewer 2: Typo:

```suggestion
* Editing the HyperConverged custom resource (CR) in your default editor
```
